src/containers/navigation-icons/AccountIcon.tsx

```diff
@@ -37,6 +37,20 @@ const AccountIcon: FC<AccountIconProps> = ({ openMenu }) => {
     defaultResponse: defaultResponses.object as UserResponse
   })
 
+  const { photo: statePhoto } = useAppSelector((state) => state.editProfile)
+
+  const getAvatarSrc = useCallback(() => {
+    if (statePhoto && statePhoto.src) {
```

Comment:
replace `statePhoto && statePhoto.src` with `statePhoto?.src` so it's more concise and easier to read